src/QueryAtomErrorResetBoundary.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Provider, atom, createStore } from 'jotai'
```

Comment:
This feels very strange.
Basically, `jotai-tanstack-query` is a package for Jotai, but it should be independent from React. Jotai atoms is framework-agnostic, like tanstack-query/core.
Please consider making `QueryErrorResetBoundary` a recipe in docs, instead of providing it as a library function.

Reply:
I thought we could do something specific for react (jotai-tanstack-query/react), but I agree with you. It's probably better off as a recipe.

Also, I think I can come up with some framework agnostic solution to reset error boundaries.

Will take a look.